src/types.ts

```diff
@@ -24,19 +11,32 @@ export interface Tags {
   };
 }
 
+export interface InvestmentOpportunity {
```

Comment:
Nit, but what do you think of ordering types by their hierarchy? So `MarketList` would be the top-most type definition in this file, etc. The current (and prior) ordering feel hard to read and I have to jump around a lot to understand the full type definition of a `MarketList`